### PR TITLE
add new line

### PR DESCRIPTION
**Pull request by kerstinlyu**

### CODE REVIEW

File: sequioa/trunk.md [hunk 1]
```diff
@@ -14,3 +14,7 @@ Greatly improve resistance to overheating in fire situation.
 ### Redcuded sap content
 
 Keeping the `bark` dry means we leave less opporunities for things to catch on fire.
+
+## water storage
+
+save water
```
Comment (kerstinlyu) [inline sequioa/trunk.md:20]: for testing

**Review by kerstinlyu — COMMENTED** [review]
comment for myself

**Review by mhoedemakers — COMMENTED** [review]
looks good!

**Comment by bkarpati** [comment]
Thanks for participating in the Git workshop. I'm going to close this PR for now and also delete the remote branch.
Please stay tuned for the follow-up.

**Pull request closed without merging**